src/mir/construction.md

```diff
@@ -1,4 +1,4 @@
-# HAIR and MIR construction
+# THIR and MIR construction
 
```

Comment:
Could you add a note that THIR used to be called HAIR? That way people that come across comments or old discussions have some context.

Reply:
Good point! Addressed via 19832d56b01e76a40f7c7aabf973483b1bb4a656.